test/end-to-end/patient/invoice/invoice.page.js

```diff
@@ -30,8 +30,8 @@ function PatientInvoicePage() {
     // set a patient with id TPA1
     findPatient.findById('PA.TPA.1');
 
-    // set the date to early this year
-    dateEditor.set(new Date('2016-02-02'));
```

Comment:
Why did this change need to take place?  Also, the date no longer matches the comment :(